### ISSUE TITLE
Position of remove button in message edit modal seems wrong

### ISSUE THREAD

**Issue by alunturner** [vector-im/element-web#24917]
### Steps to reproduce

1. Where are you starting? What can you see?
2. What do you click?
3. More steps…

### Will you send logs?

Yes

**Comment by luixxiul** [comment]
It looks the remove button (and also `View Source` button in developer mode) is based on the existing `MessageActionBar` component (you could refer from [here](https://github.com/matrix-org/matrix-react-sdk/blob/75f31defd635a4849b656852b8f21feca9a7ca8d/src/components/views/messages/EditHistoryMessage.tsx#L130-L151)), so IMHO this would need a design input for a better UX (by adding a new component for the dialog, for example).

**Comment by luixxiul** [comment]
For now, enabling highlight of messages on hover by emulating EventTile on the main timeline could improve UX a bit.

https://user-images.githubusercontent.com/3362943/227518749-93cb43fe-27b2-4104-8f73-80adce65e69c.mp4

**Comment by andybalaam** [comment]
I agree that moving the Remove button would be ideal here, but I think that https://github.com/matrix-org/matrix-react-sdk/pull/11674 is probably good enough, and this dialog is rarely-used enough that I don't want to try and spend designers' time on improving it right now, so I'm happy to say this is "fixed" by that PR.
